### ISSUE TITLE
Chrome's Modification Limit Barrier

### ISSUE THREAD

**Issue by ModisR** [ratdoux/easyreading#5]
Wiggling around the proportion slider too much causes the extension to crash with the error message that it's making too many modifications in a short space of time. To sort this out, we may have to walk back my instant preview feature and bring back ratdoux's submit button.

**Comment by ratdoux** [comment]
Looking into using setTimeout for it

**Comment by ModisR** [comment]
Ah, nice one, mate. So you adding like 1-2 secs cool down period between input change and applying it to the page?

**Comment by ratdoux** [comment]
Exactly, haven't had the chance to figure out what the maximum api count is but it would be just at that limit.

**Comment by ModisR** [comment]
I tried using `setTimeout` but it seems that setting timeouts itself counts towards the limit so, as part of my next update, I'll just change the `input` events back to `change` events like you made them initially.
